diagnostics-app/src/app/app.component.html

```diff
@@ -5,3 +5,6 @@
 -->
 
 <h1>ChromeOS Diagnostics App</h1>
+<button (click)="toggleTheme()">
+  {{ theme === "dark" ? "Enable Light Mode" : "Enable Dark Mode" }}
```

Comment:
I'd just use one text, e.g. "Enable Dark Mode", and when the user toggles the button on, dark mode should be applied, another press would turn off the dark mode and toggle the button off. WDYT (What do you think)?

Reply:
This is to be be replaced by a sun/moon icon in the header in next PR. So we can leave this as is for now.

Reply:
Sounds good. Will that icon have a "hover over" text? This will make it more user-friendly.

Reply:
Yes I'll add that.